x-pack/plugins/alerting/server/task_runner/rule_loader.ts

```diff
@@ -71,21 +72,30 @@ export function validateRule<Params extends RuleTypeParams>(
   if (!enabled) {
     throw new ErrorWithReason(
       RuleExecutionStatusErrorReasons.Disabled,
-      new Error(`Rule failed to execute because rule ran after it was disabled.`)
+      new Error(`Rule failed to execute because rule ran after it was disabled.`),
+      TaskErrorSource.USER
```

Comment:
This is the weird edge case where a rule is disabled WHILE it's running.  But can also happen if there are "zombie" alerting task docs from a 7.x upgrade.  In either case, seems like marking these as "framework" source would be better.